<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>하나만 선택 하시오</string>
   </property>
   <widget class="QCheckBox" name="checkBox_A1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>자장면</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_A2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>짬뽕</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="checkBox_B1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>탕수육</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_B2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>양장피</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_B3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>팔보채</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
